tests/jerry/es.next/array-pattern.js

```diff
@@ -329,33 +329,6 @@ function __createIterableObject (arr, methods) {
   assert (x.prop === "42");
 }) ();
 
-mustThrow ("var iter = __createIterableObject([], "
```

Comment:
So these these should not throw error now? Would it make sense to adapt the tests to expect "no error"?